controllers/release/adapter_test.go

```diff
@@ -532,12 +533,154 @@ var _ = Describe("Release adapter", Ordered, func() {
 				},
 			})
 
-			result, err := adapter.EnsureReleaseIsProcessed()
+			result, err := adapter.EnsureManagedPipelineIsProcessed()
 			Expect(!result.RequeueRequest && !result.CancelRequest).To(BeTrue())
 			Expect(err).NotTo(HaveOccurred())
 			Expect(adapter.release.IsProcessing()).To(BeTrue())
 
-			pipelineRun, err := adapter.loader.GetManagedReleasePipelineRun(adapter.ctx, adapter.client, adapter.release)
+			pipelineRun, err := adapter.loader.GetReleasePipelineRun(adapter.ctx, adapter.client, adapter.release)
+			Expect(pipelineRun).NotTo(BeNil())
+			Expect(err).NotTo(HaveOccurred())
+			Expect(adapter.client.Delete(adapter.ctx, pipelineRun)).To(Succeed())
+		})
+	})
+
+	When("EnsureTenantPipelineIsProcessed is called", func() {
```

Comment:
Can you add a test for continue processing occurs when releaseplan.spec.pipeline == nil?

Reply:
It actually looks like this is tested because you didn't change the the default releaseplan in this file to have a pipeline, but it is not obvious. Can you make it obvious? Basically I am looking for 2 versions of the test on line 570 (should continue if the PipelineRun exists and the release processing has started). One for if the RP has no pipeline, one for if the RP has a pipeline but the pipelinerun already exists

Reply:
test added.